Mathlib/Geometry/Manifold/Instances/Sphere.lean

```diff
@@ -382,7 +382,7 @@ theorem stereographic'_target {n : ℕ} [Fact (finrank ℝ E = n + 1)] (v : sphe
 
 /-- The unit sphere in an `n + 1`-dimensional inner product space `E` is a charted space
 modelled on the Euclidean space of dimension `n`. -/
-instance chartedSpace {n : ℕ} [Fact (finrank ℝ E = n + 1)] :
+instance Metric.sphere.instChartedSpace {n : ℕ} [Fact (finrank ℝ E = n + 1)] :
```

Comment:
Or should this be as follows?
```suggestion
instance EuclideanSpace.instChartedSpaceSphere {n : ℕ} [Fact (finrank ℝ E = n + 1)] :
```

Reply:
Many things can be modelled on `EuclideanSpace`, so I feel this should be named after the manifold, `Metric.sphere`.